manifests/init.pp

```diff
@@ -48,6 +48,7 @@
   $sshd_config_allowusers              = [],
   $sshd_config_denygroups              = [],
   $sshd_config_denyusers               = [],
+  $sshd_config_maxauthtries            = 6,
```

Comment:
Got it too ! I'll use directly '6'. But thanks for the advice.

Thanks a lot for your help. Sorry for the mistakes that I made. Tell me if there is something else.

I'll push again my commit.